docs/az/docs/index.md

```diff
@@ -0,0 +1,467 @@
+
```

Comment:
I believe a while ago we agreed that initial translation file (index.md) should be a separate pull request to just the setup for the docs.

Reply:
Yes, this pull request just contains a new language setup, not index.md translations

Reply:
Can't you just remove the index.md all together?

Reply:
You mean there is no need for the untranslated version of index.md too? Okay I will remove it

Reply:
Yes